spec/jobs/threat_metrix_js_verification_job_spec.rb

```diff
@@ -0,0 +1,253 @@
+require 'rails_helper'
+
+RSpec.describe ThreatMetrixJsVerificationJob, type: :job do
+  let(:proofing_device_profiling_collecting_enabled) { true }
+  let(:threatmetrix_org_id) { 'ABCD1234' }
+  let(:threatmetrix_session_id) { 'some-session-id' }
+  let(:threatmetrix_signing_certificate) do
+    <<~END
+      -----BEGIN CERTIFICATE-----
+      MIIEqjCCApICCQC8hXt9uhVWFjANBgkqhkiG9w0BAQsFADAXMRUwEwYDVQQDDAxz
+      aWduaW5nLXRlc3QwHhcNMjIwOTA5MTYwMDAxWhcNMzIwOTA2MTYwMDAxWjAXMRUw
+      EwYDVQQDDAxzaWduaW5nLXRlc3QwggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIK
+      AoICAQDTpZRdHAmkpi0EoeNB9x/FVmsc5WxETvMKvOB06O7CDYrf9SZklXjq8R8N
+      uT7el+BoM+6auVeTD+uYbyasUSGjWB83BH16q85Oy1YQxpGZitSuQctRdgs8Idsb
+      TvP9csREbC6GMlsqcgh1Qe07565SYyVF/DruwqKL28p5RtCPTtgkfxVl7Nnf8IMp
+      d1ILtTpzlYzbnSKvkXZwx1PSrXsviXhoYxlzTfbIOMDvCCEAhspGdOPHJJAT6dd4
+      iSVsLpHwyEG6pDlEIZlcE43S7tyVLiTDMORiSEYxbbLJhNWb8o+g0rOs/w3wnn/O
+      w5PdFImeGGCpH/5dBraiKR/kGdw48BNzvU0yhdZfYe4jtKZJSQ3140n8EypCXMaR
+      jl/dHMl/BcW4a3nMS+t768yL7pA1lvFs6gpf4fFxySHsZq0C3EiO6nWTOD1zg0qR
+      sfSwVZLVZF5PQNkXqDMG7gBPWGqHx6qtUtE95w3m0BC99dFekKsgH3AhFKjzoI/e
+      bB7o82am48cEWKxBzjllIaEFI+g9zHhqgZX677zMGi2MxCGiznFTRaT5ZUDSBBxa
+      F1Vba86He7PgU35925cL1MbbaSJ8LTpC3Clr2MfQAsXtZ4HjCxpriKgid0sxur/L
+      OWB7ZTe1hgmim9dWEaRqqELMyTZYcidjWdyljOGqXfDyde1DVQIDAQABMA0GCSqG
+      SIb3DQEBCwUAA4ICAQCegU8gVLeIDvvhDmJdx5IxDqtic820wFE9s4BT7N73Ik3f
+      ywwIHbSR5BSWaIyaKYKzZ5iouJxNZ4IppFYnYPVWrWGqm96V1yMhDpS2hnbsrZaA
+      EirY5DqXga9pfxDNb83gD4FD2tZoK75fzVgwdqnBlzOaGXy4Dgf9yef5Ok1jy/+p
+      750nh2uXClF6yRtB1bxUnOt54LbWInoFr+S/hDR3WJlmF33bjOEliDZ1h1R/i7I9
+      E7vchrzDc0HqiRvkUvHoE0lZX38d6l2Lb/agctVgIeKC+6p8hOdUa4dCseAcOkwV
+      FPUOWsyPnauDiKc3EKN9pUPdmn9BD9vVl5FuTTUkQPZF9dbnV392JU7nHBn8DGgo
+      99QyeXA3+qaEqVSbTYWtBqLtWyhyr/OaB+Y2TmP4c+Hdr/8pydIUluMF594e4BUz
+      L4GLfB4bdbqw6SKTAQIASq4Dlj0vLlp1U0axPzz1eH8TEtRSF5XOmjXLlI1SlaJB
+      4RpVU19RifIdFR3wSMe2Tv+SVjvbNhCRcH/glTDjownThbK73Uy9K5EbFCcxDGzm
+      k2Tzqgpbs3GNpq4BdBSxA2UExosz5EAbcxhEI2/YiH6378F/cWLepGjCVPmuHm3r
+      XemyOx/wRbYuSXPibNE3/Qw8xi9b4voyyJPPWUus+qJpgQCuMkH9UiOx65e/2g==
+      -----END CERTIFICATE-----
+    END
+  end
+
+  let(:threatmetrix_private_key) do
```

Comment:
NB: This is not a sensitive private key, just a random one I generated locally

Reply:
maybe we could re-use code like this to generate a fake pair so we don't leave in something that looks private? (sometimes automated checks flag things that look like private in our soruce)

https://github.com/18F/identity-fake-server/blob/3181adbba3df7f273dbd6766e0be4fc67468d1bb/spec/aamva_spec.rb#L38-L64

Reply:
looks like we have that same code in this repo already, sort of, https://github.com/18F/identity-idp/blob/c62f569ba319641f6e69f52c2e9b6064a9a21a27/spec/support/aamva_fixtures.rb#L14-L43